tools/GlobalToolShimGenerator/ShimGeneratorInner.targets

```diff
@@ -0,0 +1,73 @@
+<!--<Project DefaultTargets="ExtractAppHostToOutput">-->
+<Project>
+  <Import Project="$(MsbuildThisFileDirectory)\..\..\build\dependencies.props" />
+
+  <!-- 
+  To call the ExtractAppHostToOutput, the following properties need to be set.
+  - AppTargetFramework              (should be netcoreapp2.1 for now).
+  - ShimOutputDirectory             (Output path for the shims.)
+  - IntermediateDirectoryForShimsBase   (Directory to output the restored shims.)
+  - RID                             (win-x86, win-x64, osx-64.)
+  - ShimTargetName                  (Name of the tool for which this shim is being generated.)
+  - PackageId                       (PackageId of the tool.)
+  - PackageVersion                  (PackageVersion being generated.)
+  - ShimTargetFileName              (Name of the file to be invoked by the shim. (Usually the $(ShimTargetName).dll))
+  -->
+
+
+
+  <UsingTask TaskName="EmbedAppNameInHost"
```

Comment:
KoreBuild already has a mechanism for building MSBuild tasks in a local repo that typically works well for these situations.

https://github.com/aspnet/BuildTools/blob/dev/docs/KoreBuild.md#repotasks

Reply:
Sean and I gave this a try and hit an issue. Since the ShimGeneratorInner.targets is called in a separate MSBuild context, the EmbedAppNameInHost task is not available in this scope. 
Do you know a way to workaround this? 
If not, we can keep this as is for now and investigate further in the dev branch.

Reply:
Did you try doing an Import? `<Import Project="$(RepositoryRoot)build\tasks\RepoTasks.task" />`

Reply:
Of course not. That would have been too obvious 😬.